collector/src/main/java/com/bitmovin/analytics/features/segmenttracking/SegmentTrackingFeatureConfig.kt

```diff
@@ -0,0 +1,7 @@
+package com.bitmovin.analytics.features.segmenttracking
+
+import com.bitmovin.analytics.features.FeatureConfig
+
+class SegmentTrackingFeatureConfig: FeatureConfig() {
+    val maxSegments: Int = 20
```

Comment:
let's start with 10 for the beginning
```suggestion
    val maxSegments: Int = 10
```